requirements/dev.txt

```diff
@@ -19,9 +21,9 @@ ddt==1.2.1
 django-nose==1.4.6
 django-waffle==0.18.0
 django==1.11.26
-djangorestframework-jwt==1.11.0
 djangorestframework==3.9.4
 docutils==0.15.2
+drf-jwt==1.13.3
```

Comment:
Was there a discussion regarding how to proceed with this update?  Just want to ensure @nasthagiri is aware and in agreement given its security requirements.  Also, I'd update the PR and commit comments with more details about this transition once it is fully decided.   Thank you.

**Additional notes:**
It looks like there were two possible routes to go here based on this post:
https://github.com/jpadilla/django-rest-framework-jwt/issues/484

Also see this post regarding the currently selected option:
https://github.com/Styria-Digital/django-rest-framework-jwt/issues/4